libs/coin-modules/coin-tron/src/logic/estimateFees.ts

```diff
@@ -1,3 +1,14 @@
-export function estimateFees(_address: string, _amount: bigint): Promise<bigint> {
-  throw Error("Not implemented yet");
+import { ACTIVATION_FEES, ACTIVATION_FEES_TRC_20 } from "./constants";
+import type { Intent, Fees } from "@ledgerhq/coin-framework/api/index";
+
+export async function estimateFees(intent: Intent): Promise<Fees> {
```

Comment:
When I am looking at the `bridge/getEstimateFees` the fess computation seems more complex.
I don't understand the `gross` notion here 🤔

Reply:
See slack message in vault channel